apypie/action.py

```diff
@@ -47,12 +47,23 @@ def find_route(self, params=None):
     def validate(self, values=None):
         self._validate(self.params, values)
 
+    def _add_to_path(self, path=None, *additions):
+        if path is None:
+            path = ''
```

Comment:
Why not default path to `''` in the parameters?

Reply:
because we pass `None` in a few cases when calling `_add_to_path`